pyannote/audio/generators/change.py

```diff
@@ -0,0 +1,96 @@
+from pyannote.audio.generators.periodic import PeriodicFeaturesMixin
+from pyannote.core import SlidingWindowFeature
+from pyannote.core import Segment
+from pyannote.core import Timeline
+from pyannote.generators.fragment import SlidingSegments
+from pyannote.generators.batch import FileBasedBatchGenerator
+from scipy.stats import zscore
+import numpy as np
+
+
+class ChangeDetectionBatchGenerator(PeriodicFeaturesMixin,
+                                            FileBasedBatchGenerator):
+
```

Comment:
Could you please add a docstring to document the class (what it generates, role and default value of all parameters)?
I know existing generators do not have a proper docstring either but this is clearly missing.